### DISCUSSION TITLE
Commandline use?

### DISCUSSION THREAD

**Discussion by rotacak** [simple64/simple64#397] [Q&A] [answered]
Hello, 

are there any commandline commands? I would like to have switches --fullscreen and --windowed, like in mupen64 but it does not work in simple64.

Thanks.

**Comment by loganmc10** [comment, marked as answer]
These are the current command line options:
```
./simple64-gui --help
Usage: ./simple64-gui [options] ROM

Options:
  -h, --help                 Displays help on commandline options.
  --help-all                 Displays help including Qt specific options.
  -v, --verbose              Verbose mode. Prints out more information to log.
  --nogui                    Disable GUI elements.
  --test <number_of_frames>  Run debug tests.

Arguments:
  ROM                        ROM to open.
```

There isn't currently a flag for fullscreen or windowed mode

**Reply by rotacak** [reply]
And where is saved fullscreen/window settings? Can't find it anywhere, in ini or registers (on windows).

**Reply by loganmc10** [reply]
It's in the Parallel-RDP settings, on windows I believe it would be %APPDATA%/roaming/mupen64plus/mupen64plus.cfg (or something similar)
